topology/store.go

```diff
@@ -32,3 +37,53 @@ type Tor struct {
 	Id         uint64 `sql:"AUTO_INCREMENT"`
 	datacenter *common.Datacenter
 }
+
+
+// Backing store
+type topoStore struct {
+	common.DbStore
+}
+
+func (topoStore topoStore) Entities() []interface{} {
+	retval := make([]interface{}, 2)
+	retval[0] = Host{}
+	retval[1] = common.Datacenter{}
+	return retval
+}
+
+func (topoStore topoStore) CreateSchemaPostProcess() error {
+	return nil
+}
+
+func (topoStore *topoStore) findHost(id uint64) (Host, error) {
+	host := Host{}
+	topoStore.DbStore.Db.Where("id = ?", id).First(&host)
+	err := common.MakeMultiError(topoStore.DbStore.Db.GetErrors())
+	if err != nil {
+		return host, err
+	}
+	return host, nil
+}
+
+func (topoStore *topoStore) listHosts() ([]Host, error) {
```

Comment:
I don't think the concept of 'datacenter' or whether an implementation of romana covers multiple data centers, is settled yet though.